### PR TITLE
[#6759] button clean up according to design clean

### PR DESCRIPTION
**Pull request by phillimorland**
should also finish the fix for the planifolio update https://github.com/liqd/adhocracy-plus/issues/2103

### CODE REVIEW

File: adhocracy-plus/templates/a4dashboard/includes/project_list_item.html [hunk 1]
```diff
@@ -88,8 +88,8 @@ <h2 class="modal-title" id="project{{ project.pk }}DeleteModal">{% blocktrans wi
             <div class="modal-footer">
                 <form action="{% url "project-delete" pk=project.pk organisation_slug=project.organisation.slug %}" method="post">
                     {% csrf_token %}
-                    <button type="submit" class="btn btn--default">{% trans 'delete' %}</button>
                     <button type="button" class="btn btn--light" data-bs-dismiss="modal">{% trans 'cancel' %}</button>
+                    <button type="submit" class="btn btn--default">{% trans 'delete' %}</button>
```
Comment (phillimorland) [inline adhocracy-plus/templates/a4dashboard/includes/project_list_item.html:92]: swapped order to align with designs

File: adhocracy-plus/templates/a4dashboard/project_list.html [hunk 2]
```diff
@@ -9,7 +9,7 @@ <h1 class="lr-bar__left">
             {% trans 'Projects' %}
         </h1>
         <div class="lr-bar__right">
-            <a href="{% url 'a4dashboard:project-create' organisation_slug=view.organisation.slug %}" class="btn btn--light">
+            <a href="{% url 'a4dashboard:project-create' organisation_slug=view.organisation.slug %}" class="btn btn--default" disabled>
```
Comment (fuzzylogic2000) [inline adhocracy-plus/templates/a4dashboard/project_list.html:12]: Why disabled?

**Review by fuzzylogic2000 — APPROVED** [review]
All the buttons I saw look great! Just one question.

File: adhocracy-plus/templates/a4dashboard/project_list.html (hunk 2 above)
Comment (phillimorland) [reply adhocracy-plus/templates/a4dashboard/project_list.html:12]: @fuzzylogic2000 I think I removed it in another commit though as can't find it now?!

Comment (fuzzylogic2000) [reply adhocracy-plus/templates/a4dashboard/project_list.html:12]: But it's still here also when I look at all changed files and not at single commits. And I saw it in the inspector when I tried it out (even though it didn't do anything).
But I can remove it after lunch - still have it checked out anyway.

Comment (phillimorland) [reply adhocracy-plus/templates/a4dashboard/project_list.html]: @fuzzylogic2000 sorry hadn't pushed my latest :(

**Pull request merged**